tests/views_tests.py

```diff
@@ -57,7 +57,7 @@ class TwilioTokenTests(BaseTest):
     def test_get_token_for_customer_by_default(self):
         # Given
         mock_capability = MagicMock()
-        mock_capability.to_jwt.return_value = 'abc123'
+        mock_capability.to_jwt.return_value = b'abc123'
```

Comment:
`views_test.py`'s `mock_capability.to_jwt.return_value` variable needed change string to byte so I changed it like below.